src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.BIO.Custom.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.Runtime.InteropServices;
+using Microsoft.Win32.SafeHandles;
+using System.Diagnostics;
+
+internal static partial class Interop
+{
+    internal static class CustomBio
```

Comment:
Yeah, so two things,
1. I was wondering about if it should be directly in the Interop or under crypto, happy to move it.
1. The Custom really came from the OpenSsl naming but the more I used it I also became unhappy. So mabye ManagedMemoryBio, but also it is very specific to Ssl if you try to use it for say reading a cert or something, well I wouldn't at this stage (it might be an option to extend it to that later). So Maybe ManagedSslBio ?